deploy/helm/sumologic/values.yaml

```diff
@@ -26,6 +26,26 @@ eventsDeployment:
       cpu: "100m"
 
 sumologic:
+  ## Setup
+
+  # If enabled, a pre-install hook will create Collector and Sources in Sumo Logic
+  setupEnabled: true
```

Comment:
We are commenting out access id, key and endpoint because they are required values with no defaults. When we reference them [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/176/files#diff-6c6e2426d7625fb2a69ef04d6b81d272R29) , we use the `required` function so that if user does not override this value on `helm install` it will error out. For example:
```
$ helm install maisie-charts/sumologic --name collection --namespace sumologic --no-crd-hook
Error: render error in "sumologic/templates/setup-job.yaml": template: sumologic/templates/setup-job.yaml:28:14: executing "sumologic/templates/setup-job.yaml" at <required "A valid .Values.sumologic.endpoint entry required!" .Values.sumologic.endpoint>: error calling required: A valid .Values.sumologic.endpoint entry required!
```

The collector name is commented out because the script has the default value, so we don't need to specify it again here